2-mandatory/3-bush-berries.js

```diff
@@ -22,7 +22,11 @@
 */
 
 function isBushSafe(berryArray) {
-  //Write your code here
+  if (berryArray.every((item) => item === "pink")) {
```

Comment:
think about how it would affect the performance if you negated the condition and used the some fn. instead

Reply:
I do not know about performance I should learn to code better